src/site/_data/vars.js

```diff
@@ -8,4 +8,5 @@ module.exports = {
   figmaLink: 'https://www.figma.com/file/JrmVrSdKCHUZBQFEB3xsjJ/Design-Library?node-id=50%3A5303&t=xG6UMcIDeLIbH1kD-1',
   markdownContactEmailLink: `[${contactEmail}](mailto:${contactEmail})`,
   noIndex: Boolean(process.env.NO_INDEX),
+  trackingId: 'G-JPYNXT8WLC',
```

Comment:
Not a fan of the hard code value also, it installs the cookie on a test, 
My first idea was to `{trackingId: process.env.CI ? 'TEST_ID' : 'G-XXXXXX'}` but then realised the release build we have `process.env.CI as truthy`.
I can put a rule in the GA dashboard to ignore localhost and d2s dev.